AWS.md

```diff
@@ -38,7 +38,20 @@ EC2
 
 VPC
 ---
-* Ensure you have added the correct [Gateway Endpoints](https://docs.aws.amazon.com/vpc/latest/privatelink/vpce-gateway.html) for the AWS services being accessed from your private subnets to avoid incurring unnecessary networking costs.
+
+* Avoid using the default VPC - see [here](https://konekti.us/post/are-you-using-the-aws-default-vpc-stop-now/#:~:text=The%20default%20VPC%20lacks%20the,for%20auditing%20and%20troubleshooting%20purposes.) for a list of reasons why this is a bad idea. 
+* Make sure the teams consider whether the account will need to be allocated a block of our IP address space to support peering (see [here](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-basics.html) for more info on AWS peering rules.
```

Comment:
Is it worth making this a requirement? Peering is tricky; you may not know you need it up front so better to plan for it anyway. (Also re point below.)